packages/cra-template/template/src/service-worker.js

```diff
@@ -0,0 +1,48 @@
+// This service worker can be customized!
+// See https://developers.google.com/web/tools/workbox/modules
+// for the list of available Workbox modules, or add any other
+// code you'd like.
+// You can also remove this file if you'd prefer not to use a
+// service worker, and the Workbox build step will be skipped.
+
+import { clientsClaim } from 'workbox-core';
+import { precacheAndRoute, createHandlerBoundToURL } from 'workbox-precaching';
+import { registerRoute } from 'workbox-routing';
+
+clientsClaim();
+
+// Precache all of the assets generated by your build process.
+// Their URLs are injected into the manifest variable below.
+// eslint-disable-next-line no-restricted-globals
```

Comment:
Maybe this should be added to a global `eslint` config? I'm not sure.

Reply:
It didn't appear to be needed when i reviewed. There were some other TypeScript errors in this file that I've resolved.

Reply:
You can see a list of the globals we include in this rule here: 

https://github.com/facebook/create-react-app/blob/abe236963c75160f905f0cd96a2abaee245ba653/packages/confusing-browser-globals/index.js

Reply:
Resolving, as the file has moved to https://github.com/cra-template/pwa/blob/master/packages/cra-template-pwa/template/src/service-worker.js